oam/install_scripts/columnstore-post-install.in

```diff
@@ -308,6 +308,8 @@ chown -R mysql:mysql /tmp/columnstore_tmp_files
 chmod 777 /tmp/columnstore_tmp_files
 chmod 777 /dev/shm
 
+ldconfig
```

Comment:
JFYI. Debian packaging that is in the server repo under ./debian has ldconfig already so rpm scripts really needs this or we ll have a duplicate.

Reply:
Good catch